addon/-private/ext/date.js

```diff
@@ -3,7 +3,7 @@
 */
 
 import Ember from 'ember';
-import { deprecate } from 'ember-data/-private/debug';
+import { deprecate } from '../debug';
```

Comment:
Does this change need to be reverted now that `uniqueAdd(filteredImports, 'ember-data/-private/debug', ['instrument']);` is back in place?